packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -365,7 +365,20 @@ export const useArbTokenBridge = (
     },
     [bridge, bridgeTokens]
   )
-  const addTokensStatic = async (arbTokenList: TokenList) => {
+
+  const removeTokensFromList = useCallback( (listID: number)=>{
+    const newBridgeTokens = {...bridgeTokens}
+    for ( let address in bridgeTokens) {
+        const token = bridgeTokens[address]
+        if(!token) continue
+        if(token.listID === listID){
+          delete newBridgeTokens[address]
+        }
+    }
+    setBridgeTokens(newBridgeTokens)
```

Comment:
don't we want to do this inside the `setBridgeTokens((prevValue) => { const newBridgeTokens = {...prevValue}` instead of `const newBridgeTokens = {...bridgeTokens}`?

Reply:
since we're strictly removing I don't think there can be a race condition

Reply:
...altho tbh, can't hurt